snap/snapcraft.yaml

```diff
@@ -0,0 +1,134 @@
+name: opensearch # you probably want to 'snapcraft register <name>'
+base: core20 # the base snap is the execution environment for this snap
+
+version: '2.2.0' # just for humans, typically '1.2+git' or '1.3.2'
+
+summary: 'OpenSearch: community-driven, Apache 2.0-licensed search and analytics suite.'
+description: |
+    OpenSearch is a community-driven, Apache 2.0-licensed open source search and 
+    analytics suite that makes it easy to ingest, search, visualize, and analyze data. 
+    Developers build with OpenSearch for use cases such as application search, 
+    log analytics, data observability, data ingestion, and more.
+
+grade: devel # must be 'stable' to release into candidate/stable channels
+
+confinement: strict # use 'strict' once you have the right plugs and slots
+
+
+assumes:
```

Comment:
I would not worry about these for now, snapcraft should be auto-adding assumes for command-chain